user_guide_src/source/general/filters.rst

```diff
@@ -59,7 +59,9 @@ This is typically used to peform redirects, like in this example::
 
         if (! $auth->isLoggedIn())
         {
-            return redirect('login');
+            $response = Services::response();
+            $response->redirect('login',$request,$response);
+            return $response;
```

Comment:
Were you encountering issues with the was listed in the docs? Using `redirect()` should be perfectly valid. It's a [helper function](https://github.com/bcit-ci/CodeIgniter4/blob/develop/system/Common.php#L751) that's always available.

Reply:
I see that hnow - I was misusing it, trying to use the redirect method in HTTP\Response. My bad, and will fix.